### PR TITLE
Updating third party docs page to include npm packages and products

### PR DESCRIPTION
**Pull request by johnsimons**
This PR updates all the scripts to generate the 3rd party list to also include ServiceControl and ServicePulse.

The 3rd party list moves from `/nservicebus/upgrades/third-party-licenses` to `/platform/third-party-licenses`.

As part of this change we also improved the diff viewing when the changes are large, now we create a `changes.patch` artifact that is easier to review and apply manually.

This also changes the layout of the page, the page changes to a simple list of all 3rd party libs that are used for all Particular software:

image

### CODE REVIEW

**Comment by johnsimons** [comment]
@PhilBastian I am not 100% sure whether the title of the page should still be "Thrid-party licenses", thoughts?

**Comment by johnsimons** [comment]
I will update the licenses text to the new URL once this is merged

**Comment by irinascurtu** [comment]
@udidahan  we need some advice here. Should we mention all the 3rd party libraries as a "Credit to these 3rd party" ? Has this a legal implication?

File: platform/third-party-licenses.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+---
+title: Third-party licenses
+summary: View the licenses for third-party libraries used by Particular Service Platform.
```
Comment (udidahan) [inline platform/third-party-licenses.md:3]:
```suggestion
summary: Licenses for third-party libraries used by the Particular Service Platform.
```

**Comment by johnsimons** [comment]
@udidahan 

> Also, just to verify, have links to the deleted files been updated appropriately?

I have[ added a redirect](https://github.com/Particular/docs.particular.net/pull/6920/files#diff-47308e8c40cb70756ec550c675587be51214055f1c1a0c9558ad2eebe9d1152bR10), and will also update all locations found in [this search](https://github.com/search?q=org%3AParticular%20%2Fnservicebus%2Fupgrades%2Fthird-party-licenses&type=code) once this is live.

**Comment by udidahan** [comment]
OK, so while I can't say that I've reviewed all the other elements on this PR, I do approve the current state of the third-party-licenses.md file.

**Pull request merged**